Makefile

```diff
@@ -8,7 +8,7 @@ SERVICES = users things http normalizer ws coap lora influxdb-writer influxdb-re
 DOCKERS = $(addprefix docker_,$(SERVICES))
 DOCKERS_DEV = $(addprefix docker_dev_,$(SERVICES))
 CGO_ENABLED ?= 0
-GOOS ?= linux
+GOOS ?= $(shell go env GOOS || echo linux)
```

Comment:
`GOOS ?= $(go env GOOS)` should also work for linux

Can you please also add this ones:
`GOARCH ?= $(go env GOARCH)`
`GOARM ?= $(go env GOARM)`

Reply:
@mandric please use these simpler checks that @manuio proposed

Reply:
Sure thing! I think it needs `shell` keyword though.

Reply:
It's working for us on Linux and OS X without this keyword. Why you need it?

Reply:
Without the shell keyword I'm pretty sure the shell does not get invoked.  It doesn't work for me without shell keyword, I just get an empty string.

Reply:
@mandric might  be that Makefile demands this. Let me re-check.

Reply:
Yes, I checked - `shell` keyword is needed, so please just add it.

Reply:
Another option that works on OSX here is just remove the `GOOS ?= linux` line.

Reply:
Here is where it was added: https://github.com/mainflux/mainflux/pull/196

Reply:
Did you tested on Windows? I don't think that this is safe. Please, add again your previous version: 
`GOOS ?= $(shell go env GOOS)`
`GOARCH ?= $(shell go env GOARCH)`
`GOARM ?= $(shell go env GOARM)`